### PR TITLE
fix: Markdownlint cleanup for RU

### PR DESCRIPTION
**Pull request by nschonni**
Last batch of cleanup

### CODE REVIEW

**Comment by SphinxKnight** [comment]
@mdn/yari-content-ru could you please have a look?

**Comment by SphinxKnight** [comment]
If there is no veto or review in the 15 coming days, I'll merge this one after this time.

File: files/ru/web/css/_colon_nth-last-child/index.md
Comment (sashasushko) [inline files/ru/web/css/_colon_nth-last-child/index.md:12]:
```suggestion
[CSS](/ru/docs/CSS) [псевдокласс](/ru/docs/Web/CSS/Псевдо-классы) `:nth-last-child(an+b)` находит элемент, имеющий `an+b-1` потомков после данной позиции в дереве документа, значение для `n` может быть положительным или нулевым, а также имеющий родительский элемент.
```

File: files/ru/web/css/_colon_nth-last-child/index.md
Comment (sashasushko) [inline files/ru/web/css/_colon_nth-last-child/index.md:21]:
```suggestion
element:nth-last-child(an + b) { стили }
```

File: files/ru/web/css/color_value/index.md
Comment (sashasushko) [inline files/ru/web/css/color_value/index.md:47]:
```suggestion
> **Примечание:** Список принятых ключевых слов претерпел много изменений в ходе эволюции CSS:
```

**Review by sashasushko — APPROVED** [review]

**Comment by nschonni** [comment]
@sashasushko could you merge these? I can create new ones for the newly converted files once these are landed

**Pull request merged**
